C/c4Base.cc

```diff
@@ -245,6 +246,7 @@ bool c4error_mayBeTransient(C4Error err) C4API {
         websocket::kCodeGoingAway,
         websocket::kCodeAbnormal,
         websocket::kCodeUnexpectedCondition,
+        kWebSocketCloseUserTransient,
```

Comment:
The error codes in WebSocketInterface.hh should be kept in sync with the ones in c4Socket.h.

Reply:
https://github.com/couchbaselabs/BLIP-Cpp/pull/20